controllers/admin/AdminCountriesController.php

```diff
@@ -206,9 +206,9 @@ public function renderForm()
                     'required' => true,
                     'hint' => $this->trans('Two -- or three -- letter ISO code (e.g. "us" for United States).', array(), 'Admin.International.Help'),
                     /* @TODO - add two lines for the hint? */
-                    /*'desc' => $this->l('Two -- or three -- letter ISO code (e.g. U.S. for United States)').'.
+                    /*'desc' => $this->trans('Two -- or three -- letter ISO code (e.g. U.S. for United States)').'.
```

Comment:
Why not using the previous string and its domain? Also, you already told me that a three-letter ISO code would not be supported by the software, we should remove it then.

Reply:
Don't worry about this line, it's commented out :)